crates/bevy_app/src/app.rs

```diff
@@ -42,12 +41,22 @@ bevy_utils::define_label!(AppLabel);
 /// }
 /// ```
 pub struct App {
+    /// The main ECS [`World`] of the [`App`].
+    /// This stores and provides access to all the main data of the application.
+    /// The systems of the [`App`] will run using this [`World`].
+    /// If additional separate [`World`]s and [`Schedule`]s are needed, you can use `sub_app`s.
```

Comment:
```suggestion
    /// If additional separate [`World`]-[`Schedule`] pairs are needed, you can use `sub_app`s.
```
I think that explicitly communicating this grouping is probably clearer.